From bd4be59fb2574b8cbcdf22a86a6cafb7b71db503 Mon Sep 17 00:00:00 2001
From: Cong Wang <xiyou.wangcong@gmail.com>
Date: Tue, 15 May 2012 14:19:57 +0800
Subject: [PATCH] check install files passed by cmdline

We should not trust cmdline passed by users, always
check the install files passed from cmdline. This restores
the old behaviour which was changed by:

	commit c6c6a08850c7e4ee211696895f7ca3864be85052
	Author: Harald Hoyer <harald@redhat.com>
	Date:   Tue Feb 7 18:26:30 2012 +0100

	    add "install_items" to dracut.conf

Reported-by: Dave Young <dyoung@redhat.com>
Signed-off-by: Cong Wang <xiyou.wangcong@gmail.com>
Cc: Harald Hoyer <harald@redhat.com>
---
 dracut.sh |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dracut.sh b/dracut.sh
index 315b965..bf9e79f 100755
--- a/dracut.sh
+++ b/dracut.sh
@@ -770,7 +770,7 @@ done
 
 if [[ $kernel_only != yes ]]; then
     for item in $install_items; do
-        dracut_install -o "$item"
+        dracut_install "$item"
     done
     unset item
 
